pkg/scheduler/apis/config/types.go

```diff
@@ -331,7 +320,7 @@ func mergePluginSets(defaultPluginSet, customPluginSet *PluginSet) *PluginSet {
 
 	enabledPlugins = append(enabledPlugins, customPluginSet.Enabled...)
```

Comment:
in line 310, do:

`var enabledPlugins []Plugin`

This will keep it `nil` until an item is appended